resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json

```
@@ -153,7 +175,9 @@
 			"m_c notices that r_c doesn't seem to care if {PRONOUN/r_c/poss} treatments hurt.",
 			"m_c overhears r_c complaining about giving herbs to cats that don't 'deserve' them.",
 			"m_c is startled at r_c's views on sharing herbs with other Clans.",
-			"m_c bristles after seeing r_c add more deathberries to {PRONOUN/r_c/poss} herb storage."
+			"m_c bristles after seeing r_c add more deathberries to {PRONOUN/r_c/poss} herb storage.",
+			"m_c stopped r_c before {PRONOUN/r_c/subject} gave the wrong herbs to another cat. The look in r_c's eyes as {PRONOUN/r_c/subject} did so scared {PRONOUN/m_c/object}...",
```

```suggestion
			"m_c stopped r_c before {PRONOUN/r_c/subject} gave the wrong herbs to another cat. The look in r_c's eyes as {PRONOUN/r_c/subject} did so scared m_c...",
```

making it clearer which cat is being referred to.  Generally, it's not a good idea to refer to two characters via pronouns one after the other.  A good rule of thumb is being sure to use a character's name before using their pronoun, this helps establish *who* the pronoun is referring to.